### ISSUE TITLE
Hide/remove/disable parentless add links?

### ISSUE THREAD

**Issue by kleintom** [bugguide/bugguide#211]
If you visit https://beta.bugguide.net/node/add/ or http://bugguide.test/node/add/ you get some default 'Add content' links created by the node module (I think):
![parentless_add](https://user-images.githubusercontent.com/632915/124474114-c2871480-dd65-11eb-8366-db8043c3d384.jpg)
But none of these are linked to parents, so should those links be hidden and disabled?

It looks like current bugguide keeps the forum topic link since that link provides a dropdown to choose the parent forum, but redirects all the other links (Book, Image, Link).  So, for example, visiting the 'add Image' link https://beta.bugguide.net/node/add/bgimage would - if the current bugguide behavior is adopted - just redirect to https://beta.bugguide.net/node/add.

**Comment by kleintom** [comment]
It might be good to keep those links until #144 is fixed since right now I think those parentless add links (where you manually enter the parent) are the only ones hooked up to add books and links(?).

**Comment by clouseau** [comment]
You are correct. During development /node/add is handy (and it's Drupal's default content adding page). But aside from Forum Topics it should be an error to add a piece of content without a parent. Even in the case of forum, we would use the UI links to guide people to add topics, not drop them at /node/add.
